frontend/app/components/input-help.tsx

```diff
@@ -2,16 +2,16 @@ import type { ComponentProps, ReactNode } from 'react';
 
 import { cn } from '~/utils/tw-utils';
 
-export interface InputHelpProps extends ComponentProps<'span'> {
+export interface InputHelpProps extends ComponentProps<'div'> {
   children: ReactNode;
   id: string;
 }
 
 export function InputHelp(props: InputHelpProps) {
   const { children, className, ...restProps } = props;
   return (
-    <span className={cn('block max-w-prose text-gray-500', className)} data-testid="input-help" {...restProps}>
+    <div className={cn('block max-w-prose text-gray-500', className)} data-testid="input-help" {...restProps}>
```

Comment:
`block` class is not required since `DIV` is a block element by default.
```suggestion
    <div className={cn('max-w-prose text-gray-500', className)} data-testid="input-help" {...restProps}>
```